packages/indexer-common/src/indexer-management/models/indexing-rule.ts

```diff
@@ -144,7 +155,7 @@ export const defineIndexingRuleModels = (sequelize: Sequelize): IndexingRuleMode
       identifier: {
         type: DataTypes.STRING,
         primaryKey: true,
-        unique: true,
+        unique: false,
```

Comment:
is this changed to false because we allow identifier to be the same across networks? if so, can there be a check where identifier+network is still unique?

Reply:
That's correct. This change is needed specifically for the `global` rule, which will now exist 1:1 with protocol networks.
So we have a global rule for `mainnet` and another global rule for `arbitrum-one`.
The check is done by the new primary key constraint (`identifier+protocolNetwork`). I believe Sequelize understand this too and may enforce this at application level.